### PR TITLE
feat: save file with utf-8 encoding to support CJK characters

### PR DESCRIPTION
**Pull request by Jeff-Tian**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
-      travis_terminate 1;
-    fi
-  - npm run lint
-
+- REPO_VERSION="$(npx -c 'echo "$npm_package_version"')"
+- echo "REPO_VERSION=$REPO_VERSION"
+- npm run build
+- TPLANT_VERSION="$(node dist/index.js -V)"
+- echo "TPLANT_VERSION=$TPLANT_VERSION"
+- NPM_PACKAGE_VERSION="$(npm view @jeff-tian/tplant version)" || NPM_PACKAGE_VERSION=0
```
Comment (bafolts) [inline .travis.yml]: When merging to master using `@jeff-tian` should be removed.

File: .travis.yml [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
+- echo "NPM_PACKAGE_VERSION=$NPM_PACKAGE_VERSION"
+- echo "TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST"
+- echo "TRAVIS_BRANCH=$TRAVIS_BRANCH"
+- |
+  if [ "$REPO_VERSION" != "$TPLANT_VERSION" ]; then
+    echo -e '\033[0;31mPackage version and commander version must be the same.';
+    travis_terminate 1;
+  fi
+- |
+  if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_BRANCH" == "fork" ] && [ "$REPO_VERSION" == "$NPM_PACKAGE_VERSION" ]; then
```
Comment (bafolts) [inline .travis.yml]: I would keep this as `master`

File: .travis.yml [hunk 3]
```diff
@@ -1,58 +1,58 @@
...
+  - stage: npm Deployment
+    if: tag IS present
+    node_js: lts/*
+    before_deploy:
+      - ls
+      - echo "$NPM_TOKEN"
+    deploy:
+      edge: true
+      provider: npm
+      email: jeff.tian@outlook.com
```
Comment (bafolts) [inline .travis.yml]: This was probably configured to merge to your NPM but will conflict with the current setup. If the goal is to merge with this `bafolts` master I would change this back.

File: test/cjk.test.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import {exec} from 'child_process';
+import * as fs from 'fs';
+import * as os from 'os';
+import DoneCallback = jest.DoneCallback;
+
+describe('Parse codes that contains CJK characters', () => {
+
+    it('generate PlantUML for classes that contains CJK characters', (done: DoneCallback) => {
+        // tslint:disable-next-line:max-line-length
+        exec('ts-node --project ./tsconfig.json ./src/index.ts -i ./test/CJK/CJK.ts --output ./output.puml', () => {
```
Comment (bafolts) [inline test/cjk.test.ts]: This test is failing intermittently when it takes longer than 5 seconds to run.

Comment (Jeff-Tian) [reply test/cjk.test.ts]: Never met it from my local machine, would increase the jest timeout to 10s fixed it?

File: .travis.yml (hunk 3 above)
Comment (Jeff-Tian) [reply .travis.yml]: It's mis-operation. I just need the feature yesterday so published one to @jeff-tian/tplant. Reverted it.

File: .travis.yml (hunk 1 above)
Comment (Jeff-Tian) [reply .travis.yml]: It's mis-operation. I just need the feature yesterday so published one to @jeff-tian/tplant. Reverted it.

File: .travis.yml (hunk 2 above)
Comment (Jeff-Tian) [reply .travis.yml]: It's mis-operation. I just need the feature yesterday so published one to @jeff-tian/tplant. Reverted it.

**Comment by Jeff-Tian** [comment]
Thank you for your review. The change of `package.json` and `.travis.yml` are mis-operation. I need the feature yesterday in a hurry so published to `@jeff-tian/tplant`. And now they are reversed. 

The intermittently failure might have been fixed by increasing the `jest timeout`.

**Pull request merged**